packages/uniforms/__tests__/BaseForm.tsx

```diff
@@ -216,37 +218,54 @@ describe('BaseForm', () => {
   });
 
   describe('when reset', () => {
-    const wrapper = mount<BaseForm<any>>(<BaseForm schema={schema} />);
+    const createWrapper = () =>
+      mount<BaseForm<any>>(
+        <BaseForm model={model} schema={schema} onSubmit={onSubmit} />
+      );
 
     it('increase `resetCount`', () => {
+      const wrapper = createWrapper();
       wrapper.instance().reset();
 
       expect(wrapper.state('resetCount')).toBe(1);
     });
+    it('sets submitted back to false', async () => {
+      const wrapper = createWrapper();
+      const instance = wrapper.instance();
+      expect(instance.getContext().submitted).toBe(false);
+      wrapper.find('form').simulate('submit');
+      expect(instance.getContext().submitted).toBe(true);
+      instance.reset();
+      expect(instance.getContext().submitted).toBe(false);
```

Comment:
i decided that reset will also reset this new property. I guess one would expect that